### ISSUE TITLE
[arm64_32] System.NotSupportedException : Stack walks are not supported on this platform

### ISSUE THREAD

**Issue by rolfbjarne** [mono/mono#13128]
## Steps to Reproduce

1. Build https://github.com/xamarin/xamarin-macios/pull/5465 locally.
2. Launch xharness, and run the `fsharp /watchOS 64-bits (ARM64_32) - device / Release` test

### Current Behavior

```
[FAIL] FSharpTest.SprintfTest : System.TypeInitializationException : The type initializer for '<StartupCode$FSharp-Core>.$SR' threw an exception.
  ----> System.NotSupportedException : Stack walks are not supported on this platform.
		--NotSupportedException
```

### Expected Behavior

No test failures

Running the `fsharp / watchOS 32-bits - device / Release` test works fine.

**Comment by lewurm** [comment]
Probably fixed by https://github.com/mono/mono/pull/14440

**Comment by rolfbjarne** [comment]
I'm not sure if this is fixed or not, now the app crashes: https://gist.github.com/rolfbjarne/ba1173c13c7bbc89369da7531714845d

This might also be related to #11875 (which is how this test fails with armv7k).

**Comment by lewurm** [comment]
@rolfbjarne what mono revision is that?

**Comment by rolfbjarne** [comment]
> @rolfbjarne what mono revision is that?

16c6ac81b39bdb26786c2fda155a0a3500961798

**Comment by imhameed** [comment]
Quick note: this is pretty low priority right now. Ping me if you think otherwise.

**Comment by SamMonoRT** [comment]
@steveisok @vargaz - please can one of you take this over.

**Comment by rolfbjarne** [comment]
I don't think we need this anymore, this only happens on watchOS, but watchOS is not supported on .NET, and legacy Xamarin is about to be out of support.
